data/_ui/interfaces.txt

```diff
@@ -1431,6 +1435,16 @@ interface "map"
 	value "max zoom" 2
 	value "min zoom" -2
 
+	anchor top
+	string "route error"
+		center 1 59
+		size 18
+		color "black"
+	string "route error"
+		center 0 58
```

Comment:
Wasn't the message a bit higher before? (haven't tested, just looking at the values)

Reply:
No, the original value was 40px measured from the top of the screen to the top of the text. The 18pt font is 36px high, and these are drawn using central coordinates, so the centre point is 58px down.